api/src/test/java/com/epam/pipeline/manager/pipeline/RunScheduleManagerTest.java

```diff
@@ -103,214 +115,240 @@ public void setUp() {
         testPipeline.setRepository(TEST_REPOSITORY);
         testPipeline.setOwner(TEST_NAME);
         testPipeline.setRepositorySsh(TEST_PIPELINE_REPO_SSH);
-        pipelineDao.createPipeline(testPipeline);
+        testPipeline.setId(RUN_ID);
 
         createPipelineRun(RUN_ID, testPipeline.getId());
+
         runConfiguration = createRunConfiguration();
+        runConfiguration.setId(2L);
 
         PipelineUserVO userVO = new PipelineUserVO();
         userVO.setUserName(USER_OWNER);
-        userManager.createUser(userVO);
 
         testRunScheduleVO = getRunScheduleVO(RunScheduledAction.PAUSE, CRON_EXPRESSION1);
         testRunScheduleVO2 = getRunScheduleVO(RunScheduledAction.RESUME, CRON_EXPRESSION2);
         testRunScheduleVO3 = getRunScheduleVO(RunScheduledAction.RUN, CRON_EXPRESSION2);
 
+        MockitoAnnotations.initMocks(this);
+
+        doReturn(mockPipelineRunScheduleProvider).when(mockScheduleProviderManager).getProvider(ScheduleType.PIPELINE_RUN);
+        doReturn(mockRunConfigurationScheduleProvider).when(mockScheduleProviderManager).getProvider(ScheduleType.RUN_CONFIGURATION);
+        doReturn(mockPipelineUser).when(mockAuthManager).getCurrentUser();
+
     }
 
+
     @Test
-    @WithMockUser(username = USER_OWNER)
-    public void testCreateRunSchedules() {
-        final List<RunSchedule> runSchedules = runScheduleManager.createSchedules(RUN_ID, ScheduleType.PIPELINE_RUN,
-                    Arrays.asList(testRunScheduleVO, testRunScheduleVO2));
-        runSchedules.forEach(this::loadAndAssertSchedule);
+    public void testCreatePipelineRunSchedules() {
+        runScheduleManager.createSchedules(RUN_ID, ScheduleType.PIPELINE_RUN,
+                Arrays.asList(testRunScheduleVO, testRunScheduleVO2));
+
+        verify(mockRunScheduler, times(2)).scheduleRunSchedule(any());
+        verify(mockRunScheduleDao, times(1)).createRunSchedules(anyList());
+        verify(mockScheduleProviderManager, times(3)).getProvider(ScheduleType.PIPELINE_RUN);
```

Comment:
I suppose we can skip this verification in all tests. It doesn't really check anything specific and strongly depends on the implementation details.

Reply:
Could you please remove similar verifications from all the tests?

```
verify(mockScheduleProviderManager, times(3)).getProvider(ScheduleType.PIPELINE_RUN);
```